frontend/src/EditTask.svelte

```diff
@@ -328,6 +328,15 @@ async function deleteTask(proceed) {
                           </button>
                         </div>
                       </div>
+                      <div class="col-md">
```

Comment:
I think that it doesn't make much sense to say that task A is a homework for students on Monday and a test for students on Tuesday. This is a global property of the task, let's set it outside the class assignments. Also we should handle the NULL case :)